web/src/components/tasks/business-formation/business/MainBusinessAddressNj.test.tsx

```diff
@@ -100,22 +100,22 @@ describe("<MainBusinessAddressNj />", () => {
     expect(screen.getByLabelText("Address zip code")).toBeInTheDocument();
   });
 
-  it("does NOT show inline errors when missing address1 as NJ", async () => {
+  it("does show inline errors when missing address1 as NJ", async () => {
```

Comment:
[dust] I think we can simplify these test names. I'd do this for all 3.
```suggestion
  it("shows inline errors when missing address1 as NJ", async () => {
```